dependencies.yaml

```diff
@@ -114,6 +138,34 @@ dependencies:
               cuda: "12.*"
             packages:
               - cuda-nvcc
+  build-py:
+    common:
+      - output_types: conda
+        packages:
+          - scikit-build-core>=0.7.0
+      - output_types: [requirements, pyproject]
+        packages:
+          - scikit-build-core[pyproject]>=0.7.0
+  build-use-libkvikio-wheel:
+    common:
+      - output_types: conda
+        packages:
+          - libkvikio==24.8.*,>=0.0.0a0
+    specific:
+      - output_types: [requirements, pyproject]
+        matrices:
+          - matrix: {cuda: "12.*"}
+            packages:
+              - libkvikio-cu12==24.8.*,>=0.0.0a0
+          - matrix: {cuda: "11.*"}
+            packages: &libkvikio_packages_cu11
+              - libkvikio-cu11==24.8.*,>=0.0.0a0
+          - {matrix: null, packages: *libkvikio_packages_cu11}
```

Comment:
For devcontainers builds, this fallback matrix should not contain the CUDA-suffixed library versions. This should be modified to instead contain `libkvikio==24.8.*,>=0.0.0a0` (no suffix), ideally using YAML anchors.

Like this:

https://github.com/rapidsai/raft/blob/8fe29836f3a7b11a61f1a5286592bdc2f4f93ee9/dependencies.yaml#L500